Tests/AppTests/PackageController+routesTests.swift

```diff
@@ -365,9 +365,12 @@ class PackageController_routesTests: AppTestCase {
 
         // MUT
         try app.test(.GET, "/owner/package/1.2.3/documentation") {
+            // triggers a redirect which then finds no archive (doesn't actually
```

Comment:
This is potentially the wrong place for this comment, or at least it should be in another place, too. This test tests the `404` state when there are no `docArchives`. 

The successful test where this really comes into play is `test_documentationRoot_redirect`

Reply:
This tests a different scenario though. The redirect tests cover the handling of a 404 from AWS, this one here is a 500, which we actually surface back.

I've updated the inputs so the root access shows the redirect and doesn't fail with a 404, because that's already covered in the redirect tests. It's now asserting that the 500 from AWS doesn't affect the root doc route and remains a redirect.